Tests/SonarQube.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -254,7 +254,7 @@ public void E2E_HasManagedAndContentFiles_VB()
             WellKnownProjectProperties preImportProperties = CreateDefaultAnalysisProperties(rootInputFolder, rootOutputFolder);
 
             // Act
-            string projectDir = CreateAndBuildSonarProject(descriptor, rootOutputFolder, preImportProperties, isVBProject:true);
```

Comment:
ProjectDescriptor already has a ProjectLanguage property that can be used to specify whether a project is a VB project or not, so any method that has a ProjectDescriptor parameter doesn't need an isVBProject parameter.